README.md

```diff
@@ -1 +1,118 @@
-# HelloMicroverse
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello microverse] <a name="about-project"></a>
+
+
+**[Hello microverse]** is a simple webpage to show Hello Microverse
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **[html and css]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+### Prerequisites 
+
+In order to run this project you need:git 
+
+N/A
+### Setup
+
+Clone this repository to your desired folder:
+
+git clone https://github.com/ammadCheema36/HelloMicroverse.git
+### Install
+
+
```

Comment:
- [ ] Please take note that having a well-documented repository is a big plus as it helps the user easily set up and use your project, so please make sure to provide instructions on how to install the project here. 😊